Service/OS/AndroidGCMNotification.php

```diff
@@ -57,14 +65,16 @@ class AndroidGCMNotification implements OSNotificationServiceInterface
     /**
      * Constructor
      *
-     * @param $apiKey
+     * @param bool         $dryRun
+     * @param string       $apiKey
      * @param bool         $useMultiCurl
      * @param int          $timeout
      * @param Logger       $logger
-     * @param AbstractCurl $client       (optional)
+     * @param AbstractCurl $client (optional)
      */
-    public function __construct($apiKey, $useMultiCurl, $timeout, $logger, AbstractCurl $client = null)
+    public function __construct($dryRun, $apiKey, $useMultiCurl, $timeout, $logger, AbstractCurl $client = null)
```

Comment:
@richsage Okay I did it in the same way as it is in the AppleNotification class (where `$sandbox` is the first parameter in the constructor) for consistency but I can change it if you really want to :smile:.